tests/sanity/test_rock.py

```diff
@@ -2,20 +2,30 @@
 # Copyright 2024 Canonical, Ltd.
 #
 
-import os
 import subprocess
 
+from k8s_test_harness.util import env_util
 
-def test_sanity():
-    image_variable = "ROCK_FLUENT_BIT"
-    entrypoint = "fluent-bit"
-    image = os.getenv(image_variable)
-    assert image is not None, f"${image_variable} is not set"
 
+def _test_sanity(image_version):
+    rock = env_util.get_build_meta_info_for_rock_version(
+        "fluent-bit", image_version, "amd64"
+    )
+    image = rock.image
+
+    entrypoint = "fluent-bit"
     docker_run = subprocess.run(
         ["docker", "run", "--rm", "--entrypoint", entrypoint, image, "--version"],
         capture_output=True,
         check=True,
         text=True,
     )
-    assert "Fluent Bit v2.1.6" in docker_run.stdout
+    assert f"Fluent Bit v{image_version}" in docker_run.stdout
+
+
+def test_fluent_bit_2_1_6():
```

Comment:
nit: we can just parametrize the test instead of having dedicated functions
https://github.com/canonical/cert-manager-rock/pull/8/files#diff-aafa91523686591db7f607be14eddefac92bd43e12c1b4e077de779fd2722e33R27

Reply:
Good point, done.